composer.json

```diff
@@ -23,5 +23,8 @@
         "branch-alias": {
             "dev-master": "2.5-dev"
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^4.8"
```

Comment:
you should at least allow 4.8 or 5.7. 4.8 is not properly supporting PHP 7, and is unmaintained.